docs/documentation/documentation.html

```diff
@@ -59,8 +59,35 @@ <h1>Handleiding</h1>
           <section id="manon-quick-start">
             <h1>Manon gebruiken binnen een project.</h1>
             <p class="warning" role="group" aria-label="let op">
-              <span>Let op:</span> Aan deze instructies wordt nog gewerkt.
-            </p>
+                <span>Let op:</span> Aan deze instructies wordt nog gewerkt.
+              </p>
+          </section>
+          <section>
+              <h2>Manon installeren (lokaal docs bekijken)</h2>
+              <p>Manon maakt gebruik <a href="https://pnpm.io/installation">pnpm</a>, een package die beschikbaar is via npm.</p>
+              <ol>
+                <li>
+                    Manon downloaden: <code>git clone https://github.com/minvws/nl-rdo-manon.git</code>
+                </li>
+                <li>
+                    Open de folder waar Manon is geïnstalleerd, bijvoorbeeld: <code>cd /manon/install/folder</code>
+                </li>
+                <li>
+                    pnpm installeren: <code>npm install pnpm </code> 
+                </li>
+                <li>
+                    Dependencies installeren: <code>pnpm install</code>
+                </li>
+                <li>
+                    Open de Manon documentatie folder, bijvoorbeeld: <code>cd /manon/install/folder/docs</code>
```

Comment:
If the user has _just_ run `pnpm install` in the root of the project, they can use `cd docs` here.

Reply:
True, just making sure it is dummy proof!